src/common/PersonTile/index.js

```diff
@@ -1,9 +1,10 @@
-import { Name, Photo, PhotoWrapper, Tile } from "./styled";
+import { Name, Photo, PhotoWrapper, Tile, StyledLink, Person, Role } from "./styled";
```

Comment:
I just noticed this lack of imports, and I was going to fix it on my branch :D so good job!